--- sssd-2.7.4/src/external/pac_responder.m4~	2022-08-26 20:34:52.000000000 +0000
+++ sssd-2.7.4/src/external/pac_responder.m4	2022-09-15 16:59:04.926940582 +0000
@@ -21,7 +21,8 @@
         Kerberos\ 5\ release\ 1.16* | \
         Kerberos\ 5\ release\ 1.17* | \
         Kerberos\ 5\ release\ 1.18* | \
-        Kerberos\ 5\ release\ 1.19*)
+        Kerberos\ 5\ release\ 1.19* | \
+        Kerberos\ 5\ release\ 1.20*)
             krb5_version_ok=yes
             AC_MSG_RESULT([yes])
             ;;
--- sssd-2.7.4/configure~	2022-08-26 20:41:10.000000000 +0000
+++ sssd-2.7.4/configure	2022-09-15 16:58:34.090711218 +0000
@@ -22422,7 +22422,8 @@
         Kerberos\ 5\ release\ 1.16* | \
         Kerberos\ 5\ release\ 1.17* | \
         Kerberos\ 5\ release\ 1.18* | \
-        Kerberos\ 5\ release\ 1.19*)
+        Kerberos\ 5\ release\ 1.19* | \
+        Kerberos\ 5\ release\ 1.20*)
             krb5_version_ok=yes
             { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 printf "%s\n" "yes" >&6; }
